examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py

```diff
@@ -138,13 +150,13 @@ def proc_eval_fn(examples):
     )
     backbone = ResNet50V2(
         include_rescaling=True,
-        # This argument gives a 2% mIoU increase
         stackwise_dilations=[1, 1, 1, 2],
         input_shape=(512, 512, 3),
         include_top=False,
         weights="imagenet",
     )
-    model = DeepLabV3(classes=21, backbone=backbone)
+    model = segmentation.__dict__[FLAGS.model_name]
```

Comment:
This needs to be synced with master

Reply:
Yup, I explicitly avoided doing that so far while we were comparing scripts/results to avoid another variable. Updating now!